src/server/scripts/Spells/spell_priest.cpp

```diff
@@ -2642,6 +2643,25 @@ class spell_pri_purge_the_wicked_dummy : public SpellScript
     }
 };
 
+// 390622 - Rhapsody
+class spell_pri_rhapsody : public AuraScript
+{
+    bool Validate(SpellInfo const* /*spellInfo*/) override
+    {
+        return ValidateSpellInfo({ SPELL_PRIEST_RHAPSODY_PROC });
+    }
+
+    void HandlePeriodic(AuraEffect const* /*aurEff*/)
+    {
+        GetTarget()->CastSpell(GetTarget(), SPELL_PRIEST_RHAPSODY_PROC, true);
```

Comment:
we dont pass true as argument for triggered spells from Auras, we pass aurEff param

Reply:
also, it's missing part of the spell: at 20 stacks it should also trigger a spell aura that highlights the spell itself, find it on the sniff.

Reply:
there isnt one